run_test.sh

```diff
@@ -178,6 +189,7 @@ while getopts pwblcsaf OPTION; do
     esac
 done
 
-if [ $OPTIND -eq 1 ]; then
```

Comment:
I fixed a small bug here.
According to help documentation, "no args" should run_all_tests but instead it displayed HELP.

Reply:
Also, bash was complaining about the ; before "then". I removed it.

Reply:
I think the documentation is off.. the original behavior is better: do nothing if no option is provided. Can we just modify the documentation?